CMakeLists.txt

```diff
@@ -510,7 +511,11 @@ add_subdirectory(./src/appMain)
 add_subdirectory(./src/plugins)
 
 add_dependencies(${PROJECT} Policy)
-add_dependencies(${PROJECT} copy_policy_library)
+if(EXTENDED_POLICY STREQUAL "EXTERNAL_PROPRIETARY")
+  add_dependencies(${PROJECT} copy_library_Policy)
+else()
+  add_dependencies(${PROJECT} copy_policy_library)
+endif()
```

Comment:
Would it not be easier to normalize the names of these targets rather than add this conditional? I say we just use the name `copy_policy_library` in both

Reply:
@jacobkeeler good idea, thank you. Fixed in https://github.com/smartdevicelink/sdl_core/pull/3730/commits/f92ec7360f0dedbdbdc1ae726f51f1cb09dbbdcf